documentation/reference/qfieldcloud/system.en.md

```diff
@@ -4,102 +4,107 @@ tx_slug: documentation_reference_qfieldcloud_system
-| COPY          | Make source path relative and create copy of the data on pull, apply delta file on push to original data source | N/A                                                                                            |
-| KEEP_EXISTING | Make source path relative and create copy of the data on pull, apply delta file on push to original data source | N/A                                                                                            |
-
-This is the behavior of QField with the layers:
-
-| Action        | File based layer                   | Not file based layer                       |
-|---------------|------------------------------------|--------------------------------------------|
-| OFFLINE       | Create and push deltafile          | N/A (it's always file based at this point) |
-| NO_ACTION     | N/A                                | Edit the online (live) database            |
-| REMOVE        | N/A (the layer is no longer there) | N/A (the layer is no longer there)         |
-| COPY          | Create and push deltafile          | N/A                                        |
-| KEEP_EXISTING | Create and push deltafile          | N/A                                        |
-
-In summary, for with QFieldCloud:
-
-- *NO_ACTION* is used for online layers that are located on a server
-  accessible via the Internet and that are modified directly by
-  QField.
-- *HYBRID* means that a geopackage will be generated on the
-  server (or directly on the desktop for file-based layers) and
-  downloaded by clients. The client will generate deltafiles of the changes.
-- *OFFLINE* is used for example to work with local databases not
-  visible by QFieldCloud which are consolidated before being
-  loaded from the desktop to the server and are not synchronized
-  with the original data by QFieldCloud.
-- *REMOVE* will simply remove the layer from the project.
-- *KEEP_EXISTENT* will not be used for QFieldCloud syncronizations.
-
-From QFieldSync it will be possible to update a project already
-loaded on QFieldCloud. In the event that the changes concern only
-styles, forms etc. but not the structure of the layers, the
-project on the server will simply be updated.
-If there are changes in the layers structure, the project will be
-reset on the server (delta files will be deleted) and for each
-client it will be necessary to download the updated version of the
-project before being able to push new changes.
-
-### QFieldCloud Project
-    Is composed of one and only one QGIS project and the possible
-    related files (e.g. geopackages, images, ...) included the offline
-    or hybrid data package.
+
+In QFieldCloud context a QGIS project refers to all data files that are required for properly functioning QGIS project.
```

Comment:
I'm not sure I understand what this sentence should mean.

In the QFieldCloud context, a *project* refers to all data files that are required for a properly functioning QGIS project.

?

Reply:
```suggestion
In the QFieldCloud context, a project refers to all data files that are required for a properly functioning QGIS project.
```